### PR TITLE
Dummy Data filling

### PR DESCRIPTION
**Pull request by rajivharlalka**
- Changed title and desciption in config.toml to dummy-data

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # The URL the site will be built for
 base_url = "https://accio-demo.netlify.app/"
 
-title = "🧙‍♂️ Vijard"
-description = "Welcome to my world of leisure"
+title = "Accio"
+description = "Spell Bound theme for Zola SSG"
```
Comment (chirag828049) [inline config.toml:5]: "A" Spell Bound theme for Zola SSG

**Review by chirag828049 — CHANGES_REQUESTED** [review]
I think we can remove the resume link altogether for dummy page. Not all will want it. What say ?

**Comment by chirag828049** [comment]
Also. The README ; )

**Comment by rajivharlalka** [comment]
Yeah Would change the readme as well , thanks for reminding...

File: config.toml (hunk 1 above)
Comment (rajivharlalka) [reply config.toml:5]: Would add the "A", for the Resume part i guess it's ok to have it , as the user needs to make his own config.toml where the links for the header needs to be mentioned, will add an if in config.toml for config.toml whose true value would add the Resume part to the header

Comment (chirag828049) [reply config.toml:5]: That's much better

**Comment by rajivharlalka** [comment]
Dekh re baba

**Comment by rajivharlalka** [comment]
Chhor shi hi hoga...

**Pull request merged**